Example/PrimerSDK/MerchantCheckoutViewController.swift

```diff
@@ -26,19 +26,19 @@ class MerchantCheckoutViewController: UIViewController {
         
         Primer.shared.delegate = self
         configurePrimer()
-//        fetchPaymentMethods()
+        fetchPaymentMethods()
     }
     
     func configurePrimer() {
-        let (finalAmount, items) = generateAmountAndOrderItems()
         let businessDetails = generateBusinessDetails()
+
         let settings = PrimerSettings(
-            amount: finalAmount,
             currency: .SEK,
             countryCode: .se,
+            klarnaSessionType: .recurringPayment,
+            klarnaPaymentDescription: "Scooter Rental",
             isFullScreenOnly: false,
-            businessDetails: businessDetails,
-            orderItems: items
+            businessDetails: businessDetails
```

Comment:
I don't think we need this for Klarna to work btw + fullScreen defaults to false so that can also be removed

Reply:
👍